### ISSUE TITLE
Export `log` as a method

### ISSUE THREAD

**Issue by demurgos** [gulpjs/fancy-log#14]
Hi,
The current exports look like this:
```javascript
log = function(){...};
module.exports = log;
module.exports.info = function(){...};
module.exports.warn = function(){...};
module.exports.error = function(){...};
```

As you can see, the exports are not consistent: `log` is the main export but other functions are methods. I recommend to add a `.log` method (`module.exports.log = log`) for consistency. This helps with a common pattern used by many Gulp plugins where they temporarily replace the `gutil.log` method by a stub. This is no longer possible with `fancy-log`. In this specific case, I find `proxy-require` or exposing the logger used by the plugin cleaner than patching a shared dependency but this kind of patching is still popular and improving the consistency is still worth it.

**Comment by phated** [comment]
@demurgos I think that's fine as long as we leave the main export and `.log` is just an alias to that.  When I was upgrading plugins, I found it better to define fancy-log as `log` and the replace their logs with better/more descriptive methods (most use `log.info` and `log.error` - were you finding differently?

**Comment by demurgos** [comment]
I agree that the main export should remain unchanged and this should just be an alias.

I also recommended to some plugins to use `log.info` as a replacement but by default I used `log` directly. I preferred to use `log` because it was a direct replacement of the old behavior.
I also tend to not use `console.info` very much personally, but I think that you are right: it should be the right log level to use most of the time.
